[32mINFO    [0m mylogger:logger.py:80 endpoints.py::def delete_all_users(...)::line 29 > DELETE > 200 > {"message":"OK"}

[32mINFO    [0m mylogger:logger.py:80 <setup> Cleared all users from DB
[32mINFO    [0m mylogger:logger.py:80 endpoints.py::def add_user(...)::line 46 > POST > 200 > {"data":"adim333","messgae":"OK"}

[32mINFO    [0m mylogger:logger.py:80 endpoints.py::def get_user(...)::line 56 > GET > 200 > {"data":{"friends":[],"playlists":[],"user_name":"adim333"},"message":"OK"}

[32mINFO    [0m mylogger:logger.py:80 endpoints.py::def add_user(...)::line 46 > POST > 200 > {"error":"user with name adim333 already exists."}